src/FESpaces/SingleFieldFESpaces.jl

```diff
@@ -300,13 +299,23 @@ function interpolate!(object, free_values,fs::SingleFieldFESpace)
     FEFunction(fs,free_values)
 end
 
+
 function _cell_vals(fs::SingleFieldFESpace,object)
   s = get_fe_dof_basis(fs)
   trian = get_triangulation(s)
   f = CellField(object,trian,DomainStyle(s))
   cell_vals = s(f)
 end
 
+function _cell_vals(V::SingleFieldFESpace, f::Interpolable)
+  fe_basis = get_fe_dof_basis(V)
+  trian = get_triangulation(V)
+  cache = return_cache(f, Point(rand(2)))
+  b = change_domain(fe_basis, ReferenceDomain(), PhysicalDomain())
+  cf = CellField(x->evaluate!(cache, f, x), trian, ReferenceDomain())
```

Comment:
This should be `PhysicalDomain` right?
even though I think has no effect since you are directly extracting the data `get_data(cf)`